devtools/check-commit-msg.py

```diff
@@ -0,0 +1,23 @@
+#!/usr/bin/env python3
+
+import sys
+import os
+import re
+from argparse import ArgumentParser
+
+parser = ArgumentParser(description="SCUMMVM check commit message hook")
+parser.add_argument("message")
+args = parser.parse_args()
+
+with open(args.message, "rt") as f:
+    text = f.read()
+
+MESSAGE_FORMAT = re.compile(r"[A-Z]+:")
```

Comment:
You may want to also accept `_` in the subsystem name, as in ff23782647ddf0155ff4aa1a6a75cf99bb3dbe8c

Also are pre-commit hooks applied on `git revert` commits? In that case we may want to also accept default revert message formats, for example 7d6b55d319d92044b87483bdc2e0acdfc0193a8a.

Reply:
I've added _ to regex, also I've checked on test branch, it does not look like they are applied, it allowed me to just revert